include/ginkgo/core/matrix/csr.hpp

```diff
@@ -151,8 +202,17 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         int64_t clac_size(const int64_t nnz) { return 0; }
     };
 
+    /**
+     * cusparse is a strategy_type which uses the sparselib csr.
+     *
+     * @note cusparse is also can be known by hip executor to make the
+     *       conversion between cuda and hip without any problem.
```

Comment:
"cusparse is also known to the hip executor which converts between cuda and hip" would be better I think.